### DISCUSSION TITLE
Accidentally pressed free trial

### DISCUSSION THREAD

**Discussion by 2010person** [community/community#42946] [Enterprise] [answered]
### Select Topic Area

Question

### Body

Hi I have a free organization and was browsing through organization subscriptions, when I accidentally clicked on GitHub Enterprise and it initiated a free trial. Is there any way to get out?

**Comment by radar-blip** [comment, marked as answer]
You can always contact [support](https://support.github.com/contact/account) to see if they can help. 😃

**Reply by 2010person** [reply]
Thanks, will do that.
